testdata/configtest.json

```diff
@@ -1,2467 +1,2379 @@
+  
```

Comment:
For proper testing of this PR, this should be reverted to test backwards compatibility between master and your PR